src/traces/sankey/attributes.js

```diff
@@ -122,7 +121,14 @@ var attrs = module.exports = overrideAll({
             role: 'style',
             description: 'Sets the thickness (in px) of the `nodes`.'
         },
-        hoverinfo: extendFlat({}, plotAttrs.hoverinfo, {flags: []}),
+        hoverinfo: {
+            valtype: 'enumerated',
+            values: ['all', 'none', 'skip'],
+            dflt: 'all',
+            role: 'info',
+            editType: 'none',
+            description: plotAttrs.hoverinfo.description
```

Comment:
I suspect this isn't going to work with https://github.com/plotly/plotly.js/blob/master/tasks/compress_attributes.js (which removes `description`, and other fields that aren't needed at run time, from attributes when enabled). Anyway you might want to modify the description to mention that this only applies to nodes (and below for links)

Reply:
fyi, the description did appear in `plotly-schema.json` after running `npm run build`.

Anyway, I modified the description as you suggested in 7a32036.